app/content/team/index.js

```diff
@@ -144,6 +147,11 @@ function Index() {
         return priority1 - priority2;
       }
 
+      //doesn't softlock in case displayName is null for some reason
+      if (!user1.name) {
+        return -1;
+      }
+
```

Comment:
Is this just a hotfix for the search functionality on the team page?

Reply:
The logic right now is pretty solid so this if is not tripped, but while I was working on this PR and messed with displayName, I managed to soft-lock myself a lot of times, so I decided to leave it.

Reply:
Fair, should we keep it in just in case then?

Reply:
ye